### ISSUE TITLE
[Feature] Are there any pause tags?

### ISSUE THREAD

**Issue by myrisc** [fishaudio/fish-speech#409]
有些句号后衔接的太快，有些就没有问题，有没有类似停顿的标签？

**Comment by EarthBugs** [comment]
同求，要是能实现这个功能就好了

**Comment by Stardust-minus** [comment]
#515 Change title to English.

**Comment by AnyaCoder** [comment]
release in 1.5 soon

**Comment by zimo2013** [comment]
<img width="731" alt="Image" src="https://github.com/user-attachments/assets/d7f24aee-f197-480c-835e-c8e5d9356cef" />
In version 1.5.0, how to use the pause tag? Can you give me a demo?

**Comment by GUUser91** [comment]
@myrisc 
@zimo2013
I found a pause tag from this link. They are (break) and (long-break). They're only available in V1.6 and S1.
https://docs.fish.audio/text-to-speech/fine-grained-control
Samples. I'm using the (long-break) tag here. I added a (long-break) tag after every sentence:
S1 Mini
https://vocaroo.com/1lONk7RCbXXF

Full 4B S1 model from the fish audio website.
https://vocaroo.com/1aJz7BIlqZxk
